stable/kube-state-metrics/templates/deployment.yaml

```diff
@@ -110,9 +116,15 @@ spec:
 {{  if .Values.collectors.storageclasses  }}
         - --collectors=storageclasses
 {{  end  }}
+{{  if .Values.collectors.validatingwebhookconfigurations  }}
+        - --collectors=validatingwebhookconfigurations
+{{  end  }}
 {{  if .Values.collectors.verticalpodautoscalers  }}
         - --collectors=verticalpodautoscalers
 {{  end  }}
+{{  if .Values.collectors.volumeattachements  }}
+        - --collectors=volumeattachements
```

Comment:
Typo here, extra "e" in "volumeattachEments'. Chart is broken if `volumeattachements` set to true.

Reply:
Thanks for pointing this out @mbarrien. Would you like to raise a PR to fix this?